cmd/brew-bundle.rb

```diff
@@ -16,6 +16,10 @@
 #:    `brew bundle exec` [command]
 #:    Run an external command in an isolated build environment.
 #:
+#:    `brew bundle list` [--all|--brews|--casks|--taps|--mas] [--file=<path>|--global]
```

Comment:
Can you add some docs into the `README.md` too?

Reply:
Done in 8130afb